server/src/utils/httpClient.js

```diff
@@ -1,6 +1,9 @@
 const axios = require('axios');
-const dotenv = require("dotenv")
-dotenv.config()
+
+if (process.env.NODE_ENV !== "production") {
```

Comment:
Seems like we want this even in production @benhsm

Reply:
I'm using this pattern because of the notion that it is better to avoid using .env files to load environment variables in production contexts, on the premise that it is a potential point of unwanted information leakage (e.g. someone accidentally checking a .env file with secrets into source control), and that there may be other, preferred mechanisms for setting environment variables depending on the execution environment used for deployment. I suppose we'll see what ends up making the most sense as we get closer to deploying this.

Reply:
Sounds good to me - for Heroku, we've set env vars within the UI - those are read in the same way this `.env` is read locally